WolvenKit/Forms/frmMain.cs

```diff
@@ -786,6 +786,36 @@ public frmCR2WDocument LoadDocument(string filename, MemoryStream memoryStream =
                             MessageBox.Show(".w2mesh file not found in model folder!" + "\n" + "Have you extracted it properly?");
                         break;
                     }*/
+                case ".w2mesh":
+                    {
+                        if (MessageBox.Show("Would you like the tool to try to open a renderer context for the w2mesh file (experimental)?", "Try to render?", MessageBoxButtons.YesNo, MessageBoxIcon.Question) == DialogResult.Yes)
```

Comment:
If someone edits a lot of files this will be annoying. A button or something would be better.